### PR TITLE
issue 70: removed LinkedText url parser entirely and removed ability …

### PR DESCRIPTION
**Pull request by pospeselr**
…to open links in a browser from Ricochet-Refresh

### CODE REVIEW

File: src/libtego_ui/ui/qml/MessageDelegate.qml [hunk 1]
```diff
@@ -257,85 +247,22 @@ Column {
...
-            property string hoveredLink: textField.hasOwnProperty('hoveredLink') ? textField.hoveredLink : ""
-            MenuItem {
-                text: (hoveredLink.length > 0 && (hoveredLink.substr(0,9).toLowerCase() == "ricochet:")) ?
-                    //: Text for context menu command to copy a ricochet contact id to clipboard
-                    qsTr("Copy ID") :
-                    //: Text for context menu command to copy a url to the clipboard
-                    qsTr("Copy Link")
-                visible: hoveredLink.length > 0
-                onTriggered: LinkedText.copyToClipboard(hoveredLink)
-            }
```
Comment (m-simonelli) [inline src/libtego_ui/ui/qml/MessageDelegate.qml:315]: Should this menu item be retained? i.e. you have one menu item that's "Copy Link" and the other is "Copy Message"

**Comment by m-simonelli** [comment]
Resolves #55

File: src/libtego_ui/ui/qml/MessageDelegate.qml (hunk 1 above)
Comment (pospeselr) [reply src/libtego_ui/ui/qml/MessageDelegate.qml:315]: This patch completely removes generating hyperlinks from URIs. The remaining menu items should be 'Copy Selection' and 'Copy Message'f

Comment (m-simonelli) [reply src/libtego_ui/ui/qml/MessageDelegate.qml:315]: Ahhh my bad

**Review by m-simonelli — APPROVED** [review]

**Pull request merged**